daemons/mrpd/msrp.c

```diff
@@ -628,6 +628,9 @@ int msrp_event(int event, struct msrp_attribute *rattrib)
 #if LOG_MSRP
 			msrp_print_debug_info(event, attrib);
 #endif
+		} else {
+			/* free this attrib if we are not interested */
+			free(rattrib);
```

Comment:
Thanks @kencarlino. You are correct about the potential double free(). While I was working on cpputest memory leak detection I generated exactly that "fault". I'm going to propose 3 steps. (1) accept this pull request, (2) I create a pull request with cpputest memory detection enabled that will detect multiple frees [I know this works because I have just had to address exactly that error - although in the test framework, not mrp source code], (3)  create a github issue to make code changes so that msrp_event() is always called with the same assumptions regarding responsibility for freeing rattrib.

Does that sound ok to you?